server/controllers/shipment_cost.js

```diff
@@ -120,9 +123,10 @@ function runRules (engine, facts, res) {
   let array
   engine.run(facts).then(triggeredEvents => {
     // engine returns a list of events with truthy conditions
-    var cost = 0
+    var cost = { cost: 0 }
```

Comment:
no me acuerdo por qué pero `var` es mala palabra, en general se usa `let`

Reply:
https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable-in-jav

Reply:
Ahh perfecto. El problema es que vengo usando var medio a lo loco. Seria parte de otro refactor para mi creo

Reply:
creo que un buen punto medio es meter la rule 'no-var' al linter, como warning asi no rompe el build (sin embargo, creo que poner "error" en vez de "warn" y correr el linter con --fix es la mejor opcion pero existe el 0.1% de probabilidad de que rompa algo ese reemplazo, como warning es suficiente para que me moleste y eventualmente haga algo al respecto)

```
{
	"extends": ["standard", "plugin:react/recommended", "plugin:jest/recommended"],
	"plugins": ["react", "jest"],
	"env": {
		"browser": true,
		"commonjs": true,
		"es6": true,
		"node": true,
		"mocha": true
	},
	"parser": "babel-eslint",
	"settings": {
		"react": {
			"version": "^16.4.2"
		}
	},
	"rules": {
		"no-var": "warn"
	}
}
```

Reply:
Perfecto. Ya fue modificado!